Subsystem/Cage

CoderLiam926 opened the pull request:
(Cage)

CoderLiam926 committed:
I added stuffing

not a lot though

CoderLiam926 committed:
I added skin and or organs to the skeleton

IT still needs more thpugh!

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Elevator

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Cage

Henyahyah committed:
Cage and Climb

I copied the same thing we did with the Coral Scorer toggling thing over to the Cage for climbing.

Magpie-3D committed:
Constants for elevator

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Elevator

CoderLiam926 committed:
Merge branch 'main' into Subsystem/CoralScoringDevice

Henyahyah committed:
stash work merging

Henyahyah committed:
readding code

Henyahyah committed:
Cage Grabber movement and controlsl

Pushing the down arrow on the controller should move the bar down to hold the cage in place. The up arrow should raise it back to its neutral position.

Co-Authored-By: RedArrowRobotics <79018739+RedArrowRobotics@users.noreply.github.com>

CoderLiam926 committed:
Update ElevatorSubsystem.java

PIG and motors

CoderLiam926 committed:
I added control inputs for the Elevator Subsystem

Like Henry wanted!

CoderLiam926 committed:
Lots of Bug Fixes for ControlInputs, RobotContainer, and CageSubsystem

I changed some inputs, deleted a repeat of code, and changed a command type to the correct type.

Henyahyah committed:
Second Controller and Renamed Buttons

I renamed the controller inputs to make more sense and added a second controller for testing purposes.

CoderLiam926 committed:
Merge branch 'Subsystem/Cage' into Subsystem/Elevator

CoderLiam926 committed:
Merge and Rename

I merged Cage into this and renamed some controls.

Henyahyah committed:
Controller Renaming and Organizing

driveStick has been renamed to driveController, and controller2 has been renamed to sysOpController.

CoderLiam926 committed:
Merge branch 'main' into Subsystem/Cage

kjrokos reviewed (approved):
I think we still need some additional work on this, but it looks like a good start that we can merge into main right now

Henyahyah committed:
Proper "real" SparkMax IDs

I changed the IDs for the SparkMax's so we can test them properly. I also capitalized dealgae --> deAlgae.

Henyahyah committed:
Removing Cage Grabber

We don't need any motors for the cage grabber, so we don't need to code anything for it.

Henyahyah committed:
Removed Grabber from Constants

I forgot/missed it here.

Jalasta20XX commented on src/main/java/frc/robot/ControlInputs.java:
```diff
@@ -8,30 +8,42 @@
     public class Triggers {
-        public final Trigger driveStickA = driveStick.a();
-        public final Trigger driveStickB = driveStick.b();
+        public final Trigger driveControllerButtonA = driveController.a();
+        public final Trigger driveControllerButtonB = driveController.b();
+        public final Trigger driveControllerButtonX = driveController.x();
+        public final Trigger driveControllerButtonY = driveController.y();
+
+        public final Trigger sysOpControllerDpadUp = sysOpController.povUp();
+        public final Trigger sysOpControllerDpadDown = sysOpController.povDown();
+        public final Trigger sysOpControllerDpadLeft = sysOpController.povLeft();
+        public final Trigger sysOpControllerDpadRight = sysOpController.povRight();
+        public final Trigger sysOpControllerLeftBumper = sysOpController.leftBumper();
+        public final Trigger sysOpControllerRightBumper = sysOpController.rightBumper();
+        public final Trigger sysOpControllerLeftTrigger = sysOpController.leftTrigger();
+        public final Trigger sysOpControllerRightTrigger = sysOpController.rightTrigger();
```
Potentially consider removing the redundant word "Controller" in all of these.

Henyahyah replied:
Done! :)

Henyahyah committed:
Removed "Controller" in variable names

Henyahyah committed:
More "Controller" Removal
